meinberlin/apps/budgeting/assets/SupportBox.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
```

Comment:
This is just the RateBox from a4, but changed according to supporting, right?

Reply:
Well I rewrote it bit to a functional component and threw out some stuff that we dont need here, but yes, the base is the rating box :)